lib/openshift_client.rb

```diff
@@ -52,5 +52,16 @@ def initialize(uri,
     def all_entities
       retrieve_all_entities(ENTITY_TYPES)
     end
+
+    def process_template(template)
```

Comment:
@zakiva shouldn't you be passing in the parameters that must be used in the template?